mule-extensions-api-xml-dsl/src/main/java/org/mule/runtime/extension/xml/dsl/api/resolver/DslSyntaxResolver.java

```diff
@@ -179,6 +183,13 @@ public void visitDictionary(DictionaryType dictionaryType) {
     return builder.build();
   }
 
+  /**
+   * @return the {@link DslResolvingContext} being used to resolve the {@link DslElementSyntax}
+   */
+  public DslResolvingContext getContext() {
```

Comment:
is it necessary to publish this? I would suppose this is for internal use only. If it's necessary to publish, then put a better name. getContext() is too generic and this is only for resolving models